pkg/awsutils/awsutils.go

```diff
@@ -62,6 +64,9 @@ const (
 
 	// UnknownInstanceType indicates that the instance type is not yet supported
 	UnknownInstanceType = "vpc ip resource(eni ip limit): unknown instance type"
+
+	// Stagger cleanup start time to avoid calling EC2 too much. Time in seconds.
+	eniCleanupStartupDelayMax = 300
```

Comment:
Can you make the value `300 * time.Second` to make the units clear instead of just saying that in a comment?

Reply:
Well, it's used in a call to rand, so it can't be a duration. It's just a random upper limit.
```
startupDelay := time.Duration(rand.Intn(eniCleanupStartupDelayMax)) * time.Second
```
Cleaned up the printing a bit.